ui/cypress/e2e/checks.test.ts

```diff
@@ -1,5 +1,8 @@
 import {Organization} from '../../src/types'
 
+// a generous commitment to delivering this page in a loaded state
+const PAGE_LOAD_SLA = 10000
```

Comment:
this just unbreaks the test locally for me

Reply:
Makes sense, gives it more time to load